### ISSUE TITLE
Support h264 decoding

### ISSUE THREAD

**Issue by jrtcppv** [microsoft/playwright#7942]
### Feature request

I am trying to use Playwright to test a video application but am getting the following page error:
```
Failed to execute 'addSourceBuffer' on 'MediaSource': The type provided ('video/mp4; codecs="avc1.64001e"') is unsupported.
```
If I understand [this page](https://www.chromium.org/audio-video) correctly this is because Chromium does not ship with an h264 codec by default, however Chrome and Edge both support h264 so we don't see this error in our app. Would it be possible to add support for h264 in the packaged Chromium or to start supporting Edge or Chrome?

**Comment by jrtcppv** [comment]
I just found [this issue](https://github.com/microsoft/playwright/issues/2367) and followed the suggestion to use `executable_path` set to Chrome and it got me past this. Sorry about that!

**Comment by luisdelarosa** [comment]
I had this problem with the default chromium that's included with Playwright version 1.31.2.

I fixed this by enabling in the playwright.config.ts:
```
    {
      name: 'chrome',
      use: { channel: 'chrome' },
    },
```

Then at the command line, I run the test with:
`npx playwright test --headed --project=chrome`
